site/server/views/ChartPage.tsx

```diff
@@ -89,12 +91,30 @@ export const ChartPage = (props: {
                     ></figure>
 
                     {post && (
-                        <div className="originReference">
-                            <a href={`/${post.slug}`}>
-                                Here is all our research and data on{" "}
-                                <strong>{post.title}</strong>.
-                                <FontAwesomeIcon icon={faArrowRight} />
-                            </a>
+                        <div className="related-research-data">
+                            <h2>All our research and data on {post.title}</h2>
+                            <div className="research">
+                                <a href={`/${post.slug}`}>
+                                    <FontAwesomeIcon icon={faFileAlt} />
+                                    Read the article
+                                </a>
+                            </div>
+                            {relatedCharts && (
```

Comment:
Maybe do a check for `relatedCharts.length` too?

Reply:
makes sense, thanks!